### PR TITLE
Contactform

### PR DESCRIPTION
**Pull request by Danny-imo**
In this pull request: we have created a contact form.
We make sure that:
Name field accepts up to 30 chars.
Email field accepts only emails.
Message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.
Collect data by using the Formspree service.
in addition to that we:
use Flexbox.
stick to the design as much as possible (e.g., font, colors, images, text, margins) using the figma
Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, where linter checks are not running. Kindly solve the merge conflict and make sure linter checks are running properly.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by adel-gu — COMMENTED** [review]
# Status: INVALID ❌ 

**Hi @ 👋🏼  **

While you made a great effort 💪🏼💪🏼💪🏼 in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that misses the linter checks probably because the linter.yml contain some issues so please consider copying the exact file from [linter-config](https://github.com/microverseinc/linters-config) and try again 👏🏼 

Screenshot from 2023-04-05 21-48-28
Screenshot from 2023-04-05 21-58-04


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -297,3 +301,116 @@
...
+  font-family: "Poppins", sans-serif;
+  align-items: center;
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  cursor: pointer;
+  color: #6070ff;
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure the border of the `Get in touch` button matches the provided Figma design.
- [ ] Kindly make sure to change the typography on the `Get in touch` button in order to match the provided Figma design. Kindly adjust your `font-weight` and `font-size`

     - ### YOUR DESIGN
     
     image

     - ### FIGMA DESIGN
    image

File: style.css [hunk 2]
```diff
@@ -297,3 +301,116 @@
...
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 779px;
+  width: 315px;
+  border-top-left-radius: 100px;
+  padding: 8px;
+  display: flex;
+  flex-direction: column;
+  margin-left: 330px;
+}
```
Comment (levy002) [inline style.css]: - [ ] Kindly make sure the contact section is aligned well on the page in order to match the provided Figma design.

- [OPTIONAL] Kindly make sure to align the rest part of your Portfolio correctly in order to match the provided Figma design.

 - ### YOUR
    screencapture-127-0-0-1-5500-index-html-2023-04-06-12_41_32
 DESIGN
 
 - ### FIGMA DESIGN
image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Danny-imo   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -297,3 +301,122 @@
...
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #172b4d;
+  padding: 15px 38px 15px 16px;
+}
```
Comment (AbelG101) [inline style.css:393]: - [ ] [OPTIONAL] Great job 👏 fixing the button styling but please make sure the spacing between the form elements looks similar to the given FIgma design as this ensures your future clients that you can implement a given design down to the last pixel.
    - Your design:
      image
    - Figma design:
      image

**Review by AbelG101 — APPROVED** [review]
# Hi @Danny-imo 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**